src/source/worker_tile.js

```diff
@@ -169,7 +169,7 @@ class WorkerTile {
                     glyphMap = result;
                     maybePrepare.call(this);
                 }
-            }, undefined, undefined, taskMetadata);
+            }, undefined, true, taskMetadata);
```

Comment:
I don't think we need to queue this on the main thread. We need to queue the response to this

Reply:
So, the intent is to make `mustQueue` force queing only on the worker thread, whether it's the task (if you're sending from the main) or the response (if you're sending from the worker).